app/styles/vendor/_material_cards.scss

```diff
@@ -153,15 +159,189 @@ $card-actions-font-size: ceil(($font-size-base * 1.04)); // ~15px
   }
 
   &.mdl-card-border {
-    border: 1px solid $card-border-color;  
+    border: 1px solid $card-border-color;
   }
 
   &.mdl-card-expand {
-    &:focus, &:hover, &:active { 
-      box-shadow: 0 4px 5px 0 rgba(0,0,0,.14), 0 1px 10px 0 rgba(0,0,0,.12), 0 2px 4px -1px rgba(0,0,0,.2);
-      transform: translate(0, -3px);      
+    &:focus,
+    &:hover,
+    &:active {
+      box-shadow: 0 4px 5px 0 rgba(0, 0, 0, .14), 0 1px 10px 0 rgba(0, 0, 0, .12), 0 2px 4px -1px rgba(0, 0, 0, .2);
+      transform: translate(0, -3px);
     }
   }
-    
 }
 
+.brand-card-container {
+  padding: 0;
+  margin: 0;
+  -ms-box-orient: horizontal;
+  display: -webkit-box;
+  display: -moz-box;
+  display: -ms-flexbox;
+  display: -moz-flex;
+  display: -webkit-flex;
+  display: flex;
+  -webkit-flex-wrap: wrap;
+  flex-wrap: wrap; //justify-content: space-between;
+  .mdl-card {
+    background-color: $gray-lightest;
+    width: 200px;
+    height: 272px;
+    overflow: visible;
+    margin: 16px;
+    -webkit-transition: width .2s, height 2s, linear .2s;
+    /* For Safari 3.1 to 6.0 */
+    transition: width .2s, height 2s, linear .2s;
+    .image-container {
+      margin-left: auto;
+      margin-right: auto;
+      margin-top: -16px;
+      height: 125px;
+      width: 125px;
+      border-radius: 5px;
+      &.image1 {
```

Comment:
what's the idea behind the colors? Could we provide a more flexible way to add colors so that a user could specify?